packages/jest-runner/src/runTest.ts

```diff
@@ -151,7 +151,7 @@ async function runTestInternal(
   const cacheFS = {[path]: testSource};
   setGlobal(environment.global, 'console', testConsole);
 
-  const runtime = new Runtime(config, environment, resolver, cacheFS, {
+  const runtime = new Runtime(config, environment, resolver, cacheFS, path, {
```

Comment:
This is a breaking change. We need to either

1. pass it in the options object
2. pass it as the last parameter

Reply:
Passed it as the last parameter